staging/src/k8s.io/kubectl/pkg/metricsutil/metrics_printer.go

```diff
@@ -59,25 +74,38 @@ func (printer *TopCmdPrinter) PrintNodeMetrics(metrics []metricsapi.NodeMetrics,
 	w := printers.GetNewTabWriter(printer.out)
 	defer w.Flush()
 
+	measuredResources := printer.measuredResources
+	nodeColumns := printer.nodeColumns
+	if slices.Contains(measuredResources, "swap") {
+		nodeColumns = append(nodeColumns, "SWAP CAPACITY(bytes)")
+		measuredResources = append(measuredResources, "swap capacity")
```

Comment:
Why you're adding `swap capacity` do `measuredResources`. You already have `swap` there, so what's it for? It only complicates your checks further in the code where you need to check for both `swap` and `swap capacity` where only `swap` (which should be a constant - see my other comment) is sufficient.

Reply:
Currently if I run with what you have here I'm getting:
```
$ kubectl top node --show-swap 
NAME                 CPU(cores)   CPU(%)   MEMORY(bytes)   MEMORY(%)   SWAP(bytes)   SWAP(%)   SWAP CAPACITY(bytes)   
kind-control-plane   86m          0%       776Mi           1%          0Mi           0%        0Mi    
```
notice you have 3(!) colums for swap, where based on my understanding `SWAP(bytes)` and `SWAP CAPACITY(bytes)` will show the exact same value, because [node has only a single value for swap](https://github.com/kubernetes/kubernetes/blob/6b8e5a9457a015e5568d7ae21b94d8a4eeb41bb3/staging/src/k8s.io/api/core/v1/types.go#L6230-L6238). So that last columns is unnecessary.